src/main/java/uk/gov/hmcts/reform/adoption/notification/SendgridService.java

```diff
@@ -60,17 +60,20 @@ public void sendEmail(CaseData caseData, String subject, DocumentType documentTy
         Content content = new Content(LOCAL_COURT_EMAIL_SENDGRID_CONTENT_TYPE, LOCAL_COURT_EMAIL_SENDGRID_CONTENT_BODY);
         Attachments attachments = new Attachments();
         //log.info("For Testing Purpose, sendgrid email sent to address: {} ", caseData.getApplicant1().getEmailAddress());
-        log.info("Sendgrid email sent to court address: {} ",caseData.getFamilyCourtEmailId());
+        log.info("Sendgrid email to be sent to court address: {} ",caseData.getFamilyCourtEmailId());
         Mail mail = new Mail(new Email(sendGridNotifyFromEmail), subject, new Email(caseData.getFamilyCourtEmailId()), content);
         AdoptionDocument adoptionDocument = caseData.getDocumentsGenerated().stream().map(item -> item.getValue())
             .filter(item -> item.getDocumentType().equals(documentType))
             .findFirst().orElse(null);
+        log.info("Adoption Document with file ID: {}", adoptionDocument.getDocumentFileId());
         final String authorisation = idamService.retrieveSystemUpdateUserDetails().getAuthToken();
         String serviceAuthorization = authTokenGenerator.generate();
+        log.info("About to call getDocumentBinary method to fetch document binary");
         if (adoptionDocument != null) {
             Resource document = caseDocumentClient.getDocumentBinary(authorisation,
                                                                      serviceAuthorization,
                                                                      UUID.fromString(adoptionDocument.getDocumentFileId())).getBody();
+            log.info("call to getDocumentBinary method successful");
```

Comment:
Added the comments as we are calling a third party API to fetch the document binary and hence sometimes it fails and with these logs it would be easier to track any failures.